components/data_prefetch/data_prefetch.tsx

```diff
@@ -52,6 +53,7 @@ export default class DataPrefetch extends React.PureComponent<Props, {}> {
             queue.add(async () => this.prefetchPosts(currentChannelId));
             await loadProfilesForSidebar();
             this.prefetchData();
+            this.props.actions.trackDMGMOpenChannels();
```

Comment:
Just to confirm, how often is this event sent out? Having extra data isn't too much of a problem since we can filter out any extra events for a given user, but we don't want to send a ridiculous amount of events

Reply:
This event is triggered on the load of the webapp alone.